odoo/orm/fields_textual.py

```diff
@@ -324,6 +324,22 @@ def update_db_column(self, model, column):
     _description_size = property(attrgetter('size'))
     _description_trim = property(attrgetter('trim'))
 
+    def get_depends(self, model):
+        depends, depends_context = super().get_depends(model)
+
+        # display_name may depend on context['lang'] (`test_lp1071710`)
+        if (
+            self.name == 'display_name'
+            and self._depends is None
+            and not self.store
+            and model._rec_name
+            and model._fields[model._rec_name].base_field.translate
+            and 'lang' not in depends_context
+        ):
```

Comment:
I don't understand the `self._depends is None` condition ? It seems pointless to me.

Reply:
It should be `self.compute`.